controllers/releaseplan/enqueue_status.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package releaseplan
+
+import (
+	"context"
+	"strings"
+
+	"github.com/redhat-appstudio/release-service/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	crtHandler "sigs.k8s.io/controller-runtime/pkg/handler"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var log = logf.Log.WithName("event_handler")
+
+// EnqueueForReleasePlanAdmissionStatus enqueues Request containing the Name and Namespace specified in the
+// Status.ReleasePlans[] of the ReleasePlanAdmission that is the source of the Event. The source of the event
+// triggers reconciliation of the parent resource.
+type EnqueueForReleasePlanAdmissionStatus struct {
+	Type schema.GroupKind
+}
+
+var _ crtHandler.EventHandler = &EnqueueForReleasePlanAdmissionStatus{}
+
+// Create implements EventHandler
+func (e *EnqueueForReleasePlanAdmissionStatus) Create(_ context.Context, evt event.CreateEvent, q workqueue.RateLimitingInterface) {
+	// A freshly created ReleasePlanAdmission won't have any ReleasePlans in its status
+}
+
+// Update implements EventHandler
+func (e *EnqueueForReleasePlanAdmissionStatus) Update(_ context.Context, evt event.UpdateEvent, q workqueue.RateLimitingInterface) {
```

Comment:
```suggestion
func (e *EnqueueForReleasePlanAdmissionStatus) Update(_ context.Context, updateEvent event.UpdateEvent, rateLimitingInterface workqueue.RateLimitingInterface) {
```

Same for others. Let's use cleaner names please.

Reply:
fixed